js/main.js

```diff
@@ -1,2 +1,24 @@
 /* global data */
 /* exported data */
+var $photoURL = document.querySelector('.photoURL');
+var $image = document.querySelector('.image');
+$photoURL.addEventListener('input', function (event) {
+  $image.setAttribute('src', $photoURL.value);
+});
+
+var inputs = {};
```

Comment:
You should move this inside the function definition for submit, it doesn't need to exist after that function ends